libraries/common/global-theme.js

```diff
@@ -6,6 +6,13 @@ export default function () {
     const favicon = document.getElementById("favicon");
     if (favicon) favicon.href = chrome.runtime.getURL("/images/icon-blue.png");
   }
+  const now = Date.now() / 1000;
+  if (now < 1743609600 && now > 1743436800 && ["settings", "popup"].includes(location.pathname.split("/")[2])) {
+    // March 31, 2025 12:00:00 GMT to April 2, 2025 12:00:00 GMT
```

Comment:
Perfect! Our April Fools have always been 45 hours long

Reply:
@Samq64 Actually isn't this off by 4 hours?

Reply:
Oops, it looks like the tool I used to generate the timestamps used local time.